lib/IMP.js

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2020 Daniel Bohm & Patrik Perger and the Bridge Hackathon Contributors
+*
+*   Use of this source code is governed by an MIT-style
+*   license that can be found in the LICENSE file or at
+*   https://opensource.org/licenses/MIT
+*/
+
+function IMPs(score1, score2) {
+    var diff = Math.abs(score1 - score2); //score 1 is the teams score on table 1, score 2 the same teams score on table 2.
+    var points;
+    if (score1 == score2) {
+        return points= [0, 0];          //first number is IMP of the team whose score we entered, score 2 is the other teams IMP
+
+    }
+    
+        if (diff >= 20 && diff <= 40) {
```

Comment:
You need to check for diff < 20 otherwise you'll score 3nt+1 versus 4M= as worth 24 imps.

Reply:
thanks, fixed